core/src/main/java/com/novoda/noplayer/mediaplayer/NoPlayerTrackInfo.java

```diff
@@ -0,0 +1,20 @@
+package com.novoda.noplayer.mediaplayer;
+
+import android.media.MediaPlayer;
+
+class NoPlayerTrackInfo {
+
+    private final MediaPlayer.TrackInfo trackInfo;
```

Comment:
Wrapper around the `MediaPlayer.TrackInfo` because it is a static class without an accessible constructor.